src/app/components/status-indicator/index.html

```diff
@@ -0,0 +1,112 @@
+        <stache-code>&lt;div class=&quot;sky-text-success&quot;&gt;Field label&lt;/div&gt;</stache-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div>::before fa-check #71bf43</div>
+      </div>
+    </div>
+  </div>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-text-warning">Warning</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class prepends a yellow warning icon to provide a compact way to indicate a "warning" status.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <stache-code>&lt;div class=&quot;sky-text-warning&quot;&gt;Field label&lt;/div&gt;</stache-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div>::before fa-warning #fbb034</div>
+      </div>
+    </div>
+  </div>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-text-danger">Danger</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class prepends a red warning icon to provide a compact way to indicate a "danger" status.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <stache-code>&lt;div class=&quot;sky-text-danger&quot;&gt;Field label&lt;/div&gt;</stache-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div>::before fa-warning #ef4044</div>
+      </div>
+    </div>
+  </div>
+	
+<stache-page-anchor>
+	UX guidelines
+</stache-page-anchor>
+	
+	<div class="sky-subsection-heading">Status importance</div>
+		<p>Some statuses are crucial to a user's task, while others are no more important than other information on the page or page area. If they are crucial to a user's task you should use a status indicator. If they are ancillary, treat them as regular text.</p>
+		<p>Just because one status in an area is crucial, does not mean they all are. For example, a list of invoices could show a mix of paid, unpaid and overdue invoices. It's likely that in that case only the overdue invoices would need a status indicator if they are the only invoices that need immediate attention and action.</p>
+			
+	<div class="sky-subsection-heading">Status indicator placement</div>
+		<p>Status indicators should be used in an inline context tied to a specific piece of data, rather than in a summary area. For example, an indicator can be placed in a grid cell or card. If you need to convey a higher-level status in a summary area, use a <a routerLink="label">label</a> instead.</p>
```

Comment:
I think this warrants a guidelines topic for status information. I think it would be useful to explicitly explain when to use a status indicator vs. a label vs. an alert vs. whatever other similar components we have.

Reply:
So, looking at the call out info guidelines, I guess we have that covered already.  Although it strikes me as possibly confusing that alerts convey status but we don't mention them in the Status section.

Reply:
We are going to be revisiting the "Call out info" info guidelines to tie the other components referenced on the page together better, but want to get these up in the interim as an incremental improvement.

Reply:
Sounds good. Thanks.